src/rustc.rs

```diff
@@ -108,6 +108,12 @@ impl Sysroot {
     }
 }
 
+impl From<PathBuf> for Src {
+    fn from(path: PathBuf) -> Self {
+        Self { path }
```

Comment:
I am not sure if we really want to permit implicit conversion via `into` here -- can you make this an inherent `Self::new` instead?